### ISSUE TITLE
type orderedmap.OrderedMap issue while installing

### ISSUE THREAD

**Issue by rehabaam** [chrusty/protoc-gen-jsonschema#19]
I am getting an error when installing the package

```
# github.com/chrusty/protoc-gen-jsonschema/internal/converter
../../github.com/chrusty/protoc-gen-jsonschema/internal/converter/types.go:68:3: cannot use make(map[string]*jsonschema.Type) (type map[string]*jsonschema.Type) as type *orderedmap.OrderedMap in field value
../../github.com/chrusty/protoc-gen-jsonschema/internal/converter/types.go:226:49: invalid operation: recursedJSONSchemaType.Properties["value"] (type *orderedmap.OrderedMap does not support indexing)
../../github.com/chrusty/protoc-gen-jsonschema/internal/converter/types.go:231:83: invalid operation: recursedJSONSchemaType.Properties["value"] (type *orderedmap.OrderedMap does not support indexing)
../../github.com/chrusty/protoc-gen-jsonschema/internal/converter/types.go:265:3: cannot use make(map[string]*jsonschema.Type) (type map[string]*jsonschema.Type) as type *orderedmap.OrderedMap in field value
../../github.com/chrusty/protoc-gen-jsonschema/internal/converter/types.go:298:28: invalid operation: jsonSchemaType.Properties[fieldDesc.GetName()] (type *orderedmap.OrderedMap does not support indexing)
../../github.com/chrusty/protoc-gen-jsonschema/internal/converter/types.go:300:29: invalid operation: jsonSchemaType.Properties[fieldDesc.GetJsonName()] (type *orderedmap.OrderedMap does not support indexing)
```

I checked the code in types.go file, and found that it is related to [jsonschema](https://github.com/alecthomas/jsonschema) project as Properties definition has changed from "map[string]*Type{}" to "*orderedmap.OrderedMap" as per this [commit](https://github.com/alecthomas/jsonschema/commit/42cb9b014a9687f68669cb5071d1bb48f81b681a).

Is there any plan for a new release to support this change?

**Comment by jlegrone** [comment]
In the mean time, removing the `-u` flag from the install command should build with pinned dependency versions:

```
go get github.com/chrusty/protoc-gen-jsonschema/cmd/protoc-gen-jsonschema && go install github.com/chrusty/protoc-gen-jsonschema/cmd/protoc-gen-jsonschema
```

**Comment by Bak3y** [comment]
Removing that flag didn't fix this issue for me, is there another workaround to try?

**Comment by rehabaam** [comment]
> Removing that flag didn't fix this issue for me, is there another workaround to try?

Yeah, it didn't work for me too. I had to generate the schema manually.

**Comment by wk8** [comment]
Should be fixed by https://github.com/chrusty/protoc-gen-jsonschema/pull/21

**Comment by chrusty** [comment]
> Removing that flag didn't fix this issue for me, is there another workaround to try?

Sorry, yes you would need to reset the repo to deal with this. The upstream JSONSchema library introduced some annoying breaking changes. @wk8 has a PR which will fix this once we merge it.

**Comment by chrusty** [comment]
@wk8 fixes are in the new release [0.8.0](https://github.com/chrusty/protoc-gen-jsonschema/releases/tag/0.8.0)

Thank you for reporting this, @rehabaam
